xtokens/src/lib.rs

```diff
@@ -254,18 +256,19 @@ pub mod module {
 			origin: OriginFor<T>,
 			currency_id: T::CurrencyId,
 			amount: T::Balance,
-			fee: T::Balance,
+			fee_id: T::CurrencyId,
```

Comment:
can you add new method instead modify existing one, otherwise it will be a major transaction version breaking change.

Reply:
absolutely